templates/documents/general_list.html

```diff
@@ -90,12 +90,12 @@ <h2>{xlt t='Documents'} <a href='interface/patient_file/summary/demographics.php
 				<fieldset>
 
 					<legend>{xlt t='Document Uploader/Viewer'}</legend>
-                    <div style="padding: 0 10px">
+                    <div class="py-0 px-10">
```

Comment:
Unfortunately there is no such thing as `px-10` as it only goes up to `px-5` in BS4.

Reply:
> Unfortunately there is no such thing as `px-10` as it only goes up to `px-5` in BS4.

i guess i need a coffee now :)